src/Decimals.jl

```diff
@@ -1,25 +1,19 @@
 # Pure Julia decimal arithmetic
 # @license MIT
 # @author jack@tinybike.net (Jack Peterson), 7/3/2014
-VERSION < v"0.7.0-beta2.199" && __precompile__()
 
 module Decimals
     import Base: ==, +, -, *, /, <, float, inv, round
 
-    if VERSION < v"0.7.0-DEV.3449"
-        import Base: normalize
-    else
-        export normalize
-    end
-
     export Decimal,
            decimal,
-           number
+           number,
+           normalize
 
     const DIGITS = 20
 
     # Numerical value: (-1)^s * c * 10^q
-    struct Decimal <: Real
+    struct Decimal <: AbstractFloat
```

Comment:
Just curious if you want to include this change with the rest of the changes.

Reply:
That change needs to be in that commit so `Decimal` can use `isequal(::AbstractFloat, ::AbstractFloat)`